airflow/providers/qubole/example_dags/example_qubole.py

```diff
@@ -65,11 +65,11 @@ def compare_result(**kwargs):
         :rtype: bool
         """
         ti = kwargs['ti']
-        qubole_result_1 = t1.get_results(ti)
-        qubole_result_2 = t2.get_results(ti)
+        qubole_result_1 = hive_show_table.get_results(ti)
+        qubole_result_2 = hive_s3_location.get_results(ti)
         return filecmp.cmp(qubole_result_1, qubole_result_2)
 
-    t1 = QuboleOperator(
+    hive_show_table = QuboleOperator(
```

Comment:
Seems as unrelated change but lets keep it

Reply:
This is a related change. The variable t1 was used in this file, but once it had one operator and then another decorator. Mypy then reported that the variable type is invalid. 
To solve this problem I had to enter more descriptive variable names.
https://github.com/apache/airflow/blob/2be7efa5c023d96f4266df67c7da2e90020e16f9%5E%5E%5E%5E/airflow/providers/qubole/example_dags/example_qubole.py#L255
https://github.com/apache/airflow/blob/2be7efa5c023d96f4266df67c7da2e90020e16f9%5E%5E%5E%5E/airflow/providers/qubole/example_dags/example_qubole.py#L72